docs/ChangeLog/20240825.md

```diff
@@ -4,7 +4,7 @@
 
 August 2024 brings another maintenance release of QMK. Almost all PRs this cycle were to do with cleanup or re-organization of existing features and code.
 
-One key item is that there are no longer any `via`-enabled keymaps in the QMK repository -- these have all transitioned to an [External Userspace](../newbs_external_userspace) repository under the VIA team's control. Going forward, all PRs to `qmk_firmware` must not contain `via` keymaps; these should instead be redirected to the the VIA team's repo: `https://github.com/the-via/qmk_userspace_via` (link placeholder until transfer is completed).
+One key item is that there are no longer any `via`-enabled keymaps in the QMK repository -- these have all transitioned to an [External Userspace](../newbs_external_userspace) repository under the VIA team's control. Going forward, all PRs to `qmk_firmware` must not contain `via` keymaps; these should instead be redirected to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository.
```

Comment:
@j-hap Thanks for updating your PR. This change seems like it was added by accident.

Reply:
mhh I have no idea how that made it in there, I just rebased onto main, I'll take it out

Reply:
Oh I see, I did a rebase, but forgot to sync my fork before 🤦‍♂️

Reply:
Looks better now ;-)